ecommerce/models.py

```diff
@@ -162,6 +162,19 @@ class Receipt(TimestampedModel):
     order = ForeignKey(Order, null=True, on_delete=CASCADE)
     data = JSONField()
 
+    @property
+    def payment_method(self):
+        """Try to guess the payment source based on the Cybersource receipt"""
```

Comment:
If for some reason nothing matches, should there be a default value (like "Credit Card" if `payment_method == "card"`)?

<img width="478" alt="Screen Shot 2020-06-15 at 3 20 36 PM" src="https://user-images.githubusercontent.com/187676/84699034-d2e00d80-af1e-11ea-95fd-78095192ad8b.png">

Reply:
If `payment_method == "card"`, there should be some text even if it's just a `|`. Should we say `Unknown` instead?

Reply:
As long as there's some text I'm fine with displaying whatever that is

Reply:
Sure, I'll put "Other"